docs/paper/dev/getting-started/project-setup.md

```diff
@@ -0,0 +1,141 @@
+---
+slug: /dev/project-setup
+---
+
+# Paper Project Setup
+
+As the Paper team primarily uses [IntelliJ IDEA](https://www.jetbrains.com/idea/), this guide will be focused on that IDE. 
```

Comment:
Would it make sense here to also state why we use gradle over maven ?
A lot of spigot users are using maven, so at least a bit of "all of papers current toolchain is implemented on top of gradle, hence we use it bla bla" comment would be nice I think.

Reply:
Added a small explanation to the use of Gradle, as well as the gradle guide to converting